fix: edit support link (#421)
CleverCloud/documentation: content/guides/pgpool.md

The various features are available in the [official Pgpool documentation](https://www.pgpool.net/docs/latest/en/html/index.html).

{{< callout type="info" >}}

> We only support the **Streaming** mode, which is the most used and recommended mode for Pgpool-II. If you need other modes or features that are not supported on Clever Cloud, please contact [Clever Cloud Support](https://console.clever-cloud.com/ticket-center-choice).

Reviewer: ```suggestion
PostreSQL add-ons only support the **Streaming** mode, which is the most used and recommended mode for Pgpool-II. If you need other modes or features that aren't supported on Clever Cloud, contact [Clever Cloud Support](https://console.clever-cloud.com/ticket-center-choice).
```

Your application will use a Unix Domain Socket to connect to Pgpool-II. Unix domain sockets are faster than TCP sockets because there is no handshake and the connection is made locally on the same machine. A special environment variable will be injected to your environment variables: `CC_PGPOOL_SOCKET_PATH`. This variable contains the path to the Unix Domain Socket you have to connect to. See [Usage](#usage) below for some examples on how to use it.

> The available Pgpool-II variables and their descriptions are available on our [environment variables reference page](https://www.clever-cloud.com/doc/reference/reference-environment-variables/).

Reviewer: ```suggestion
The available Pgpool-II variables and their descriptions are available on the [environment variables reference page](https://www.clever-cloud.com/doc/reference/reference-environment-variables/).
```

Use Pgpool-II to distribute the load and separate **WRITE** and **READ** queries between your PostgreSQL servers.

> When using the **Streaming** mode, it's not Pgpool-II that manages the replication of your databases. Replication must be set up on the PostgreSQL side, otherwise known as **binary**, **Hot Standby** or **Streaming** replication. To schedule this procedure, you can contact [Clever Cloud Support](https://console.clever-cloud.com/ticket-center-choice) or make a request via our [sales form](https://www.clever-cloud.com/en/contact-sales).

Reviewer: Wasn't sure of who must do the set up, tried to specify it here.

```suggestion
When using the **Streaming** mode, it's not Pgpool-II that manages the replication of your databases. Clever Cloud engineers must manually set up the replication on the PostgreSQL side, otherwise known as **binary**, **Hot Standby** or **Streaming** replication. To schedule this procedure, you can contact [Clever Cloud Support](https://console.clever-cloud.com/ticket-center-choice) or make a request via our [sales form](https://www.clever-cloud.com/en/contact-sales).
```
